src/components/Shared/StandardButton.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+const props = defineProps({
+  type: {
+    type: String,
+    default: 'button'
+  },
+  text: String,
+  render: Function,
+  activated: Boolean,
```

Comment:
Hi, what is activated?

Reply:
it is a state where the button is toggled like check-in => checked-in

Reply:
by doing this, the component is not standardised anymore. If want to change the text then pass in new text from parent. and to have the button look as selected, the term is called active. tailwind has the active class css
https://tailwindcss.com/docs/hover-focus-and-other-states